tracpro/msgs/tests/test_views.py

```diff
@@ -90,6 +90,23 @@ def test_conversation(self):
         self.assertEqual(list(response.context['object_list']),
                          [self.inboxmsg2, self.inboxmsg1])
 
+    def test_send_response(self):
+        # User can send a response to a conversation.  The page POSTs their response data.
+        url = reverse('msgs.inboxmessage_conversation', args=[self.contact1.pk])
+
+        # a task will try to fetch the latest messages from rapidpro
+        self.mock_temba_client.get_messages.return_value = []
+
+        # Post a short message as a response.
+        response = self.url_post('unicef', url, data={'text': 'Now is the time'}, follow=True)
+        self.assertRedirects(
+            response,
+            reverse('msgs.inboxmessage_conversation', kwargs=dict(contact_id=self.contact1.pk)),
```

Comment:
This reverse call looks almost identical to the `url` value in line 95. Can we reuse it here? Or is it not really identical.

Reply:
I hadn't noticed that, thanks. I'll do that.